src/components/Fields/Contract/Contract.tsx

```diff
@@ -8,12 +9,20 @@ type ContractOptions = { children: string; value: string }[];
 export const Contract = () => {
   const { program } = useParams();
   const contracts = useContracts(program ?? "");
+  const form = useFormContext<PCOL>();
+  const { field } = useController<PCOL>({
+    name: "DODAAC",
+    control: form.control,
+  });
+
   const options: ContractOptions = [];
   contracts.data?.forEach((item) => {
-    options.push({
-      children: item.Title,
-      value: item.Title,
-    });
+    if (field.value.includes(item.DODAAC)) {
```

Comment:
This partially addresses #10. Suggest possibly driving validation to ensure that Contract does in fact belong to DODAAC, as now, if they select one DODAAC and Contract. Then decide they selected the wrong DODAAC and change iit -- then it keeps an invalid Contract selected and thinks all is well. Another option (or in addition) would be to blank out the Contract upon them changing DODAACs. Could even auto populate Contract if only 1 exists for that DODAAC -- although use case may be REALLY low.